var/spack/repos/builtin/packages/genesis/package.py

```diff
@@ -0,0 +1,112 @@
+    and their complexes.
+    """
+
+    homepage = "https://www.r-ccs.riken.jp/labs/cbrt/"
+    url = "https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2"
+    git = "https://github.com/genesis-release-r-ccs/genesis-2.0.git"
+
+    version("2.0b", branch="master")
+    version(
+        "1.5.1",
+        sha256="62a453a573c36779484b4ffed2dfa56ea03dfe1308d631b33ef03f733259b3ac",
+        url="https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2",
+    )
+
+    resource(
+        when="@1.5.1",
+        name="user_guide",
+        url="https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2019/10/GENESIS-1.4.0.pdf",
+        sha256="da2c3f8bfa1e93adb992d3cfce09fb45d8d447a94f9a4f884ac834ea7279b9c7",
+        expand=False,
+        placement="doc",
+    )
+
+    variant("openmp", default=True, description="Enable OpenMP.")
+    variant("single", default=False, description="Enable single precision.")
+    variant("hmdisk", default=False, description="Enable huge molecule on hard disk.")
+
+    conflicts("%apple-clang", when="+openmp")
+
+    depends_on("autoconf", type="build", when="@1.5.1 %fj")
+    depends_on("autoconf", type="build", when="@2.0b")
+    depends_on("automake", type="build", when="@1.5.1 %fj")
+    depends_on("automake", type="build", when="@2.0b")
+    depends_on("libtool", type="build", when="@1.5.1 %fj")
+    depends_on("libtool", type="build", when="@2.0b")
+    depends_on("m4", type="build", when="@1.5.1 %fj")
+    depends_on("m4", type="build", when="@2.0b")
+
+    depends_on("mpi", type=("build", "run"))
+    depends_on("lapack")
+    depends_on("python@2.7.18", type=("build", "run"), when="@2.0b")
+
+    patch("fj_compiler.patch", when="@2.0b %fj")
+    patch("fj_compiler_1.5.1.patch", when="@1.5.1 %fj")
+
+    parallel = False
+
+    @property
+    def force_autoreconf(self):
+        # Run autoreconf due to build system patch
+        return self.spec.satisfies("@1.5.1 %fj")
+
+    def configure_args(self):
+        spec = self.spec
+        options = []
+        options.extend(self.enable_or_disable("openmp"))
+        options.extend(self.enable_or_disable("single"))
+        options.extend(self.enable_or_disable("hmdisk"))
+        if "+cuda" in spec:
+            options.append("--enable-gpu")
+            options.append("--with-cuda=%s" % spec["cuda"].prefix)
+        else:
+            options.append("--disable-gpu")
+        if spec.target == "a64fx" and self.spec.satisfies("@2.0b %fj"):
+            options.append("--host=Fugaku")
+        return options
+
+    def setup_build_environment(self, env):
+        env.set("FC", self.spec["mpi"].mpifc, force=True)
+        env.set("F77", self.spec["mpi"].mpif77, force=True)
+        env.set("CC", self.spec["mpi"].mpicc, force=True)
+        env.set("CXX", self.spec["mpi"].mpicxx, force=True)
+        env.set("LAPACK_LIBS", "{0}".format(self.spec["lapack"].libs.ld_flags))
```

Comment:
```suggestion
        env.set("LAPACK_LIBS", self.spec["lapack"].libs.ld_flags)
```
These are equivalent in Python

Reply:
I fixed that according to your advice .